client/shared/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -148,18 +150,7 @@ trait Client[F[_]] {
     */
   def get[A](s: String)(f: Response[F] => F[A]): F[A]
 
-  @deprecated("use public method with MonadCancelThrow instead", since = "0.23.7")
-  private[client] def translate[G[_]: Async](
-      fk: F ~> G
-  )(gK: G ~> F)(implicit F: MonadCancelThrow[F]): Client[G] = translateImpl(fk)(gK)
-
-  /** Translates the effect type of this client from F to G
-    */
-  def translate[G[_]: MonadCancelThrow](
-      fk: F ~> G
-  )(gK: G ~> F)(implicit F: MonadCancelThrow[F]): Client[G] = translateImpl(fk)(gK)
-
-  /** As [[expectOptionOr]], but defined in terms of [[cats.data.OptionT]]. */
+  /** As [[#expectOptionOr]], but defined in terms of [[cats.data.OptionT]]. */
```

Comment:
A little fix of scaladoc
```suggestion
  /** As [[expectOptionOr]], but defined in terms of [[cats.data.OptionT]]. */
```

Reply:
Oops, I thought that changed the other direction.